lib/main.dart

```diff
@@ -26,19 +26,30 @@ class MyHomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return DefaultTextStyle.merge(
-      style: const TextStyle(
-        decoration: TextDecoration.none,
-        color: Color.fromRGBO(255, 255, 255, 1.0),
-        fontSize: 16.0,
-        fontWeight: FontWeight.w400,
-        fontFamily: '-apple-system',
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: const Color(0xff040724),
+        toolbarHeight: 72.0,
+        leadingWidth: 86.0,
+        leading: Padding(
+          padding: const EdgeInsets.only(left: 16.0),
+          child: Image.asset('images/logo.png'),
```

Comment:
What about setting filterQuality to FilterQuality.high?
When I tested it, it shows better quality.

Reply:
It makes quite a difference. 
 
![image](https://user-images.githubusercontent.com/60090790/183362773-ba51906c-57c8-4173-b2f5-2937afe7cf1c.png)

Reply:
I'll setting FilterQuality.high. 
Thanks, I didn't notice 👍

Reply:
OK. As soon as you push the commit, I'll approve this PR.